.php-cs-fixer.php

```diff
@@ -1,28 +1,39 @@
 <?php
 
 /*
- * This document has been generated with
- * https://mlocati.github.io/php-cs-fixer-configurator/#version:3.13.1|configurator
- * you can change this configuration by importing this file.
+ * This document has been initially generated with
+ * https://mlocati.github.io/php-cs-fixer-configurator/#version:3.5.0|configurator
+ * and then adapted be our needs
```

Comment:
```suggestion
 * and then adapted to our needs
```

Reply:
Ich mach das mal kurz auf Deutsch: Das ist lustig, dass du das anmerkst, weil das in der Vorlage auch schon so falsch drinnen ist. Mir ist das vollkommen egal. Wenn du das geändert haben willst, mach das gerne :D